README.md

```diff
@@ -427,6 +452,16 @@ The API Supports a number of options to configure the read
    </td>
    <td>Write</td>
   </tr>
+  <tr valign="top">
+   <td><code>writePath</code>
+     </td>
+       <td>Used in Java connector (for Spark versions greater than 2.4) to control the write mode. If value <code>direct</code> is
+       supplied then connector writes to BigQuery directly without first writing to GCS and the value <code>indirect</code> writes
+       via GCS.
+       <br/>(Optional, defaults to <code>direct</code>)
+     </td>
+   <td>Write (scope: Jave connector)</td>
```

Comment:
Instead of "Scope", please mention it is only supported by the Spark 2.4 dedicated connector

Reply:
Done